.gitattributes

```diff
@@ -9,6 +9,8 @@ docs/**/*.png filter=lfs diff=lfs merge=lfs -text
 docs/**/*.jpg filter=lfs diff=lfs merge=lfs -text
 docs/**/*.tif filter=lfs diff=lfs merge=lfs -text
 docs/**/*.fm filter=lfs diff=lfs merge=lfs -text
+docs/**/*.pdf filter=lfs diff=lfs merge=lfs -text
```

Comment:
@rusu24edward ... FWIW...I don't think we want *all* the PDFs in docs to NOT be handled by git lfs, do we? I mean, I thought the goal was to ensure PDFs that are used in the on-line docs to work in which case it would be something like `src/docs/**/*.pdf` having the effect of *restricting* the excepted files to just that sub-tree.

Reply:
.gitattributes is a list of files that are handled with git-lfs. My understanding from our conversation in #4555 is that sphinx files, which live in /src/docs/, should not be git-lfs. So that is why I did not add them here. However, it seemed that we still wanted to enforce pdfs within /docs (not /src/docs) to be git lfs tracked, so that is why I have added this line.

Reply:
> .gitattributes is a list of files that are handled with git-lfs.

Doh! 😮  I was understanding the negated logic here...that .gitattributes held the *exceptions* to the rule. Ok, now I understand.